### ISSUE TITLE
Central place to view all documentation and examples

### ISSUE THREAD

**Issue by v0dro** [SciRuby/daru#363]
Since daru is growing pretty fast, it becomes cumbersome to update all the documentation consisting of comments in the code, notebooks (stored in another repo) and blog posts.

We should probably create a github page for daru `daru.github.io` where all this can exist in a centralized manner. The code docs will be updated from the codebase, blog posts can take the form of articles (or maybe a separate 'blogs' section in the website) and notebooks can be a walk-through of any given feature (stored in the daru repo itself).

**Comment by info-rchitect** [comment]
great idea!  would there be blog posts on using daru with iruby?

**Comment by v0dro** [comment]
Yeah. You can see notebooks in the README as an example as of now too.
